hs_core/middleware.py

```diff
@@ -0,0 +1,32 @@
+from django.http import HttpResponse
+from django.conf import settings
+from django.utils.deprecation import MiddlewareMixin
+
+
+class HSClientMiddleware(MiddlewareMixin):
+    """Default middleware for checking if a request is made from deprecated versions of hsclient
+
+    HS>3.0 is only compatible with hsclient>=1.1.5
+    The version of hsclient is passed in the user agent request header
+    """
+
+    def process_response(self, request, response):
+        # https://github.com/hydroshare/hsclient/blob/1.1.5/hsclient/hydroshare.py#L1331
```

Comment:
This code line in hsclient is probably missing f-string formatting?

Reply:
good catch thank you. Fixed in hsclient 1.1.6